### ISSUE TITLE
error on startup with custom set comment strings

### ISSUE THREAD

**Issue by rudotriton** [b3nj5m1n/kommentary#44]
I started getting an error on startup:

`kommentary/lua/kommentary/config.lua:215: attempt to index local 'defaults' (a function value)`

I have the following settings which seem to cause this:

```lua
kommentary.configure_language("lua", {
  single_line_comment_string = "--",
  prefer_single_line_comments = true,
})

kommentary.configure_language("vimwiki", {
  multi_line_comment_strings = { "<!--", "-->" },
  prefer_multi_line_comments = true,
})
```

Do I need to set them differently now after the latest updates?

**Comment by b3nj5m1n** [comment]
Hi, that's a bug, thanks for pointing this out. I'll fix it as soon as possible.

**Comment by b3nj5m1n** [comment]
Should be fixed with f0b6d75df0a263fc849b0860dc8a27f4bed743db, if you could check it out and let me know that would be great.

**Comment by rudotriton** [comment]
Perfect, thanks 🙌
